### ISSUE TITLE
Atari 800xl games support

### ISSUE THREAD

**Issue by hmontes** [OpenEmu/Atari800-Core#3]
Any plan to support atari800xl atr/exec files? Atari 800 has awesome games that doesn't have the atari 5200 system.

Thanks

**Comment by clobber** [comment]
Yes, I'd like to have a release that focuses on computer systems (including the Atari 8-bit line) barring any usability issues.
